### ISSUE TITLE
Edit links identified only by text

### ISSUE THREAD

**Issue by HaSistrunk** [RockefellerArchiveCenter/aurora#261]
Links should be bold, underlined, or have a color contrast ratio of at least 3:1 (measured text to text). See [WCAG 2.0 Technique G183](https://www.w3.org/TR/2008/NOTE-WCAG20-TECHS-20081211/G183)

**Comment by helrond** [comment]
Closed by #271
